openprescribing/measure_definitions/carbon_salbutamol.json

```diff
@@ -0,0 +1,81 @@
+    "<p>Data on carbon emissions per salbutamol inhaler can be viewed in the ",
+    "<a href='https://docs.google.com/spreadsheets/d/1eyk2iqNM_4sobDzevZWWcRR00O2tmJO5HwdFZLjhfa4'>",
+    "table here</a>.</p>"
+  ],
+  "tags": [
+    "core",
+    "respiratory"
+  ],
+  "url": null,
+  "is_percentage": false,
+  "is_cost_based": false,
+  "low_is_good": true,
+  "numerator_type": "custom",
+  "numerator_columns": [
+    "SUM(",
+    "CASE WHEN bnf_code = '0301011R0AAAPAP' THEN quantity * 25.24 --Salbutamol 100micrograms/dose inhaler CFC free \n",
+    "WHEN bnf_code = '0301011R0AABUBU' THEN quantity * 11.79 --Salbutamol CFC free breath actuated inhaler 100 micrograms \n",
+    "WHEN bnf_code = '0301011R0AAAQAQ' THEN quantity * 0.58 --Salbutamol 200micrograms/dose dry powder inhaler derived from Ventolin Accuhaler \n",
+    "WHEN bnf_code = '0301011R0BEAHAQ' THEN quantity * 0.58 --Ventolin 200micrograms/dose Accuhaler \n",
+    "WHEN bnf_code = '0301011R0BEAIAP' THEN quantity * 28.26 --Ventolin 100micrograms/dose Evohaler \n", 
+    "WHEN bnf_code = '0301011R0BIAFAP' THEN quantity * 11.95 --Salamol 100micrograms/dose inhaler CFC free (Teva) \n",
+    "WHEN bnf_code = '0301011R0BIAHAP' THEN quantity * 11.95 --Salamol 100micrograms/dose inhaler CFC free (Arrow) \n",
+    "WHEN bnf_code = '0301011R0BIAGBU' THEN quantity * 12.08 --Salamol 100micrograms/dose Easi-Breathe inhaler \n",
+    "WHEN bnf_code = '0301011R0BMAAAP' THEN quantity * 9.72 --Airomir 100micrograms/dose inhaler \n",
+    "WHEN bnf_code = '0301011R0BMABBU' THEN quantity * 9.72 --Airomir Autohaler \n",
+    "WHEN bnf_code = '0301011R0BWAABZ' THEN quantity * 0.62 --Easyhaler Salbutamol sulfate 100micrograms/dose dry pdr inh \n",
+    "WHEN bnf_code = '0301011R0BWABAQ' THEN quantity * 0.62 --Easyhaler Salbutamol sulfate 200micrograms/dose dry pdr inh \n",
+    "WHEN bnf_code = '0301011R0BXAACB' THEN quantity * 3.75 --Salbulin Novolizer 100micrograms/dose inhalation powder \n",
+    "WHEN bnf_code = '0301011R0BXAACB' THEN quantity * 3.75 --Salbulin Novolizer 100micrograms/dose inhalation powder refill \n",
+    "ELSE 0 end) AS numerator"
+  ],
+  "numerator_from": "{hscic}.normalised_prescribing",
+  "numerator_where": [
+    "bnf_code IN ('0301011R0AAAPAP','0301011R0AABUBU','0301011R0AAAQAQ','0301011R0BEAHAQ','0301011R0BEAIAP','0301011R0BIAFAP','0301011R0BIAHAP','0301011R0BIAGBU','0301011R0BMAAAP','0301011R0BMABBU','0301011R0BWAABZ','0301011R0BWABAQ','0301011R0BXAACB','0301011R0BXAACB')"
+  ],
+  "denominator_type": "custom",
+  "denominator_columns": [
+    "SUM(quantity) AS denominator"
+  ],
+  "denominator_from": "{hscic}.normalised_prescribing",
+  "denominator_where": [
+    "bnf_code IN ('0301011R0AAAPAP','0301011R0AABUBU','0301011R0AAAQAQ','0301011R0BEAHAQ','0301011R0BEAIAP','0301011R0BIAFAP','0301011R0BIAHAP','0301011R0BIAGBU','0301011R0BMAAAP','0301011R0BMABBU','0301011R0BWAABZ','0301011R0BWABAQ','0301011R0BXAACB','0301011R0BXAACB')"
```

Comment:
Last code in line 61 similar to above


I think this should be `0301011R0BXABCC`